man/landscape-config.txt

```diff
@@ -83,6 +83,12 @@ OPTIONS
   --is-registered          Exit with code 0 (success) if client
                            is registered else returns 5. Display
```

Comment:
should probably update this to match the "real" behaviour:
```
Exit with code 0 (success) if client
has sent registration request else returns 5.
Display registration sent info.
(NOTE: use --actively-registered to detect if
registration has been accepted server-side)
```
something like that.